Tools/Scripts/generate-dirty-files

```diff
@@ -135,14 +154,15 @@ def find_project_results(args, project, file_list, results_data, issue_to_report
 
     for type, count in bug_counts.items():
         print(f'    {type}: {count}')
-    return results_data
+    return results_data, results_for_upload
```

Comment:
Is there any advantage to return the same object we got passed as a parameter? And then we totally ignore the return value, seemingly.

Reply:
Weird leftovers from old behavior, probably. Thanks for being a fresh set of eyes!